monkey/common/agent_configuration/agent_configuration.py

```diff
@@ -15,6 +15,14 @@ class AgentConfiguration(MutableInfectionMonkeyBaseModel):
         "seconds)",
         default=30,
     )
-    credential_collectors: Tuple[PluginConfiguration, ...] = Field(title="Credential collectors")
-    payloads: Dict[str, Dict] = Field(title="Payloads")
-    propagation: PropagationConfiguration = Field(title="Propagation")
+    credential_collectors: Tuple[PluginConfiguration, ...] = Field(
+        title="Credential collectors",
+        description="Configure options related to the credential collection stage of the attack",
```

Comment:
Suggested change: 
Configure options for the attack’s credential collection stage